### PR TITLE
Docs/nb section3 raptor

### LINKED ISSUE #2865: Documentation: Raptor Notebook
focus on structure right now

### PR DESCRIPTION
**Pull request by andre-merzky**
Raptor is not easily amenable to notebooks, as one needs to define the master and worker scripts in separate files.  This notebook now contains a very minimal example: it creates a minimal master script on the fly and uses the default worker script.  Otherwise though it is complete and functional.  I would like to discuss the approach before expanding. 

This fixes #2865

### CODE REVIEW

**Comment by mturilli** [comment]
@okilic1 , different colors for note and warning are the standard for rendering notebooks in html. I would keep them separated.
@andre-merzky ping on the other comments?

**Comment by okilic1** [comment]
> @okilic1 , different colors for note and warning are the standard for rendering notebooks in html. I would keep them separated.

@mturilli  What I meant was the same as the other documents. Currently, the note on the top of the page is not in a box.


> @andre-merzky ping on the other comments?

**Review by mturilli — COMMENTED** [review]
Addressing Özgur comment

**Comment by andre-merzky** [comment]
> Some suggestions about RAPTOR.ipynb:

Thanks for your comments, @okilic1 !  I tried to address them now.  The custom paths should be removed in the final version - they only serve to make testing faster.

File: docs/source/tutorials/raptor.ipynb [hunk 1]
```diff
@@ -0,0 +1,242 @@
+{
```
Comment (mturilli) [inline docs/source/tutorials/raptor.ipynb:1]: <p>"The default RAPTOR Worker has a <code>test</code> function defined which we can use for proof of concept:" -&gt; The default RAPTOR Worker has a built-in <code>test</code> function which you can use for testing purpose:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (mturilli) [reply docs/source/tutorials/raptor.ipynb:1]: If you agree and address the above, I think it is ready to be merged. Thank you!

Comment (andre-merzky) [reply docs/source/tutorials/raptor.ipynb:1]: Done!

**Pull request merged**